plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -252,19 +245,16 @@ evaluateScriptRestricting verbose cmdata budget p args = swap $ runWriter @LogOu
 -- also returns the used budget.
 evaluateScriptCounting
     :: VerboseMode     -- ^ Whether to produce log output
-    -> CostModelParams -- ^ The cost model to use
+    -> EvaluationContext -- ^ The cost model that should already be synced to the latest ProtocolCostModel
```

Comment:
Again, is it really the latest one?

Reply:
I should rephrase, I didn't meant latest protocol version, but most recently updated costmodel params coming from a particular protocol